Makefile

```diff
@@ -24,8 +25,11 @@ test-sanity:
 test-integration:
 	ansible-test integration --docker -v --color --retry-on-error --python $(PYTHON_VERSION) --continue-on-error --diff --coverage $(?TEST_ARGS)
 
-test-molecule:
-	molecule test
+converge-molecule:
+	molecule converge -- -v --tags $(ANSIBLE_TAGS)
```

Comment:
Can we parameterize the verbosity here, defaulting to `-v`?

Reply:
yes can do. Thanks for the feedback